### ISSUE TITLE
Enable use of string renderer on extra_config

### ISSUE THREAD

**Issue by marcelofabri** [thoughtbot/liftoff#248]
If that's something that makes sense, I can send a PR later.

**Comment by grantjk** [comment]
Makes sense for us. We just ran into this today trying to use `<%= project_name %>` in an `extra_config` setting. It'd be awesome if that was evaluated out like in other places in the configuration file.

**Comment by jakecraige** [comment]
Good idea, perhaps this could be extended so that all config can be run through the string renderer?
